conclusion.tex

```diff
@@ -6,35 +6,22 @@ \section{Future Work}
 mapper works as promised by integrating it into existing VTK applications such
 as ParaView and Slicer. Once these tasks are complete, we have some ideas on new
 features we would like to add to this mapper (outlined below). We would also
-like to solicit feedback from folks using the VTK volume mappers. What features
-do you need? Drop us a line at kitware@kitware.com, and let us know.
+like to solicit feedback from the community using the VTK volume mappers. What
+features do you need? Drop us a line at kitware@kitware.com, and let us know.
```

Comment:
I think it probably not best to ask for feedback in paper, I am in favor of removing the kitware@kitware.com reference.